File: Begginers/C++/senha_fixa_1114.cpp
#include <stdio.h>
 
int main() {
 
    int senha;

    while(true) {
        scanf("%d", &senha);
        
        if (senha == 2002) {
        printf("Acesso Permitido\n"); 
        break;
    }
    else {
        printf("Senha Invalida\n");
    }
    }
    
    return 0;
}

File: Begginers/Java/senha_fixa_1114.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
       public static void main(String[] args) throws IOException {
              Scanner sc = new Scanner(System.in);

              int x;

              while (true) {
                     x = sc.nextInt();

                     if (x != 2002) {
                            System.out.println("Senha Invalida");
                     }
                     else {
                            System.out.println("Acesso Permitido");
                            break;
                     }
              }

              sc.close();

       }
}
